index.html

```diff
@@ -35,11 +35,16 @@ <h2>Experience</h2>
             <p> During my virtual classes in front-end development, I had the opportunity to acquire a comprehensive set of skills and deepen my understanding of the field. Over the course of the program, I delved into HTML, CSS, and JavaScript, gaining proficiency in crafting responsive and visually appealing user interfaces. The virtual class provided a platform for hands-on learning, where I successfully completed various projects that challenged me to implement complex layouts, optimize website performance, and integrate interactive features. Collaborating with classmates and instructors in the virtual environment allowed me to sharpen my communication and teamwork skills, as we leveraged online forums and video conferences to exchange ideas and provide valuable feedback. I encountered and overcame numerous obstacles throughout the program, developing strong problem-solving abilities and an adaptive mindset. By continuously seeking additional resources beyond the virtual classes, such as online tutorials and coding challenges, I demonstrated my commitment to lifelong learning and staying updated in this rapidly evolving field. Overall, the virtual class experience in front-end development has equipped me with the knowledge, skills, and confidence to pursue real-world projects and thrive in the industry.</p>
     </section>
     <!-- Skills -->
-        <section id="skills">
-            <h3>Skills</h3>
-                <ul>
-                </ul>
-        </section>
+    <section id="skills">
+        <h3>Skills</h3>
+        <ul>
+        </ul>
+    </section>
+    <section id="projects">
+        <h2>Projects</h2>
+            <ul></ul>
```

Comment:
```suggestion
        <ul></ul>
```
Indentation should be even.